third_party/move/move-model/src/model.rs

```diff
@@ -5102,6 +5108,30 @@ impl<'env> FunctionEnv<'env> {
         set
     }
 
+    /// Get used functions including transitive inline functions
+    pub fn get_used_functions_with_transitive_inline(&self) -> BTreeSet<QualifiedId<FunId>> {
+        if let Some(trans_used) = &*self.data.used_functions_with_transitive_inline.borrow() {
+            return trans_used.clone();
+        }
+
+        let mut set = BTreeSet::new();
+        let mut reachable_funcs = VecDeque::new();
+        reachable_funcs.push_back(self.clone());
+
+        while !reachable_funcs.is_empty() {
```

Comment:
Could we do `while let Some(fnc) = reachable_funcs.pop_front() { .. }` instead (to avoid unwrap and two different calls)?